APItokenmanager/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Assuming you already have a Dockerfile setup for Django
+FROM python:3.8
```

Comment:
`python:3.8` is a bit old at this point -- any reason that this version is the base layer?

Reply:
No special reason. I will update it.

Reply:
I have updated to python:3.10